### PR TITLE
Inspector: remove feature flag and angular "Export CSV" menu option

### PR DESCRIPTION
**Pull request by ryantxu**
For 7.0, all panels will have a consistent export data feature using the panel inspector.  This PR removes the feature flag and the angular modal that only worked on the graph panel.

### CODE REVIEW

**Review by torkelo — APPROVED** [review]
Looks ok, question is should we wait for the transforms before merging this? 

should inspect/export csv apply some default field order correction so that time field is first? wonder where we should handle that :)

**Comment by ryantxu** [comment]
Lets merge this after #23206

**Pull request merged**
